Adding some notes about editor SLAs (#1144)
openjournals/joss: docs/submitting.md

- An Associate Editor-in-Chief will carry out an initial check of your submission, and proceed to assign a handling editor.
- The handling editor will assign two or more JOSS reviewers, and the review will be carried out in the [JOSS reviews repository](https://github.com/openjournals/joss-reviews).
- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository. 
- **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors expected to respond in a timely fashion.

> - Authors and reviewers are asked to be patient when waiting for a response from an editor. Please allow two weeks for an editor to respond to a question before prompting them for further action.

Reviewer: I feel like 2 weeks is really slow, unless an editor is out of the office.  I could imagine this leading our average review time to increase a lot, if editors decide that they can generally wait this long.

> - Authors and reviewers are asked to be patient when waiting for a response from an editor. Please allow two weeks for an editor to respond to a question before prompting them for further action.

Author: ```suggestion
- Authors and reviewers are asked to be patient when waiting for a response from an editor. Please allow a week for an editor to respond to a question before prompting them for further action.
```

- Upon successful completion of the review, authors will make a tagged release of the software, and deposit a copy of the repository with a data-archiving service such as [Zenodo](https://zenodo.org/) or [figshare](https://figshare.com/), get a DOI for the archive, and update the review issue thread with the version number and DOI.
- After we assign a DOI for your accepted JOSS paper, its metadata is deposited with CrossRef and listed on the JOSS website.
- The review issue will be closed, and an automatic tweet from [@JOSS_TheOJ](https://twitter.com/JOSS_TheOJ) will announce it!

If you want to learn more details about the review process, take a look at the [reviewer guidelines](reviewer_guidelines).

### A note about 

## Confidential requests

Please write admin@theoj.org with confidential matters such as retraction requests, report of misconduct, and retroactive author name changes.

In case of a name change, the DOI will be unchanged and the paper will be updated without publishing a correction notice or notifying co-authors.

> JOSS will also update CrosrRef metadata.

Reviewer: ```suggestion
JOSS will also update Crossref metadata.
```

- An Associate Editor-in-Chief will carry out an initial check of your submission, and proceed to assign a handling editor.
- The handling editor will assign two or more JOSS reviewers, and the review will be carried out in the [JOSS reviews repository](https://github.com/openjournals/joss-reviews).
- Authors will respond to reviewer-raised issues (if any are raised) on the submission repository's issue tracker. Reviewer and editor contributions, like any other contributions, should be acknowledged in the repository. 

> - **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors expected to respond in a timely fashion.

Author: ```suggestion
- **JOSS reviews are iterative and conversational in nature.** Reviewers are encouraged to post comments/questions/suggestions in the review thread as they arise, and authors are expected to respond in a timely fashion.
```
